org.kde.skanlite.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "org.kde.skanlite",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "skanlite",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11"
```

Comment:
```suggestion
        "--socket=fallback-x11"
```

Reply:
My bad. I remember making that change earlier, but it was not committed. 
Thanks for the patience.